components/button/test/button-icon.visual-diff.js

```diff
@@ -35,8 +34,8 @@ describe('d2l-button-icon', () => {
 					const selector = `#${entry.category}`;
 
 					if (name === 'hover') await page.hover(selector);
-					else if (name === 'focus') await page.$eval(selector, (elem) => forceFocusVisible(elem));
-					else if (name === 'click') await page.$eval(selector, (elem) => elem.focus());
+					else if (name === 'focus') await focusWithKeyboard(page, selector);
+					else if (name === 'click') await focus(page, selector);
```

Comment:
I don't love how we're using `focus()` for "click"... but at the same time maybe it's just the test name that's incorrect. Technically this is testing the "focus styles when focus is placed using the mouse".